operator/midstream/iib.json

```diff
@@ -0,0 +1,19 @@
+{
+  "iibs": {
+    "4.10": "545335",
+    "4.11": "545336",
+    "4.12": "545337",
+    "4.13": "545338",
+    "4.14": "545339"
+  },
+  "operator": {
+    "version": "1.0.0",
+    "channel": "rhacs-1.0"
+  },
+  "ppc64le": {
```

Comment:
This is using support package 2.5.0 which is associated with ACS 4.1. We are now on 2.6.0.

Moreover, the probe downloads for 2.6.0 are now working, so we can remove the support package setup from this PR.

Reply:
@robbycochran Can this be merged now?